### PR TITLE
Remove target balance

### PR DESCRIPTION
**Pull request by jhrcek**
- UI for target balance removed
- Added new warning when restoring old strategies which had this setting enabled:

Screenshot from 2019-09-01 10-42-15

### CODE REVIEW

File: docs/StrategyFormatChangelog.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # Změny formátu strategie
 
+## Robozonky 5.4.0
+
+Po zavedení nové evropské směrnice [PSD2](https://cs.wikipedia.org/wiki/Sm%C4%9Brnice_PSD2) odstranilo
```
Comment (jhrcek) [inline docs/StrategyFormatChangelog.md]: @triceo please check this user facing message (it's the markdown document linked from the warning shown above). Do you consider this understandable for end users?

File: docs/StrategyFormatChangelog.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+Zonky ze svých programových rozhraní informace o disponibilním zůstatku.
+Tyto informace RoboZonky používal pro funkci investování na základě disponibilního zůstaku.
+Tuto funkci jsme proto byli nuceni odstranit a spolu s ní i sekci `Obecná nastavení > Disponibilní zůstatek` ve webovém konfiguračním nástroji.
+Pokud jste měli ve své strategii zapnuto nastavení `Investovat pouze pokud disponibilní zůstatek přesáhne X Kč.`,
+ bylo z vaší strategie odstraněno. RoboZonky tedy odteď bude investovat vždy bez ohledu na disponibilní zůstatek.
+
 ## Robozonky 5.3.0
 
-Robozonky odstranilo podporu pro potvrzování investic mobilem.
+Robozonky odstranil podporu pro potvrzování investic mobilem.
```
Comment (jhrcek) [inline docs/StrategyFormatChangelog.md]: Please clarify if I should talk about RoboZonky as him or her :)

Comment (triceo) [reply docs/StrategyFormatChangelog.md]: I consider RoboZonky to be a him. :-)

File: docs/StrategyFormatChangelog.md (hunk 1 above)
Comment (triceo) [reply docs/StrategyFormatChangelog.md]: This is very clear. However, the screenshot contains a typo - "nastavenou" instead of "nastaveno".

**Pull request merged**